### ISSUE TITLE
docker-compose up exit with code 1

### ISSUE THREAD

**Issue by mhetreramesh** [ruddra/docker-django#12]
My fresh installation not working at all.

Whenever I do 

> docker-compose up

 get following error & exit. If I check in the browser I get ngnix Bad Gateway error.

```
dz01     | Operations to perform:
dz01     |   Apply all migrations: admin, auth, contenttypes, sessions
dz01     | Running migrations:
dz01     |   Applying contenttypes.0001_initial... OK
dz01     |   Applying auth.0001_initial... OK
dz01     |   Applying admin.0001_initial... OK
dz01     |   Applying admin.0002_logentry_remove_auto_add... OK
dz01     |   Applying contenttypes.0002_remove_content_type_name... OK
dz01     |   Applying auth.0002_alter_permission_name_max_length... OK
dz01     |   Applying auth.0003_alter_user_email_max_length... OK
dz01     |   Applying auth.0004_alter_user_username_opts... OK
dz01     |   Applying auth.0005_alter_user_last_login_null... OK
dz01     |   Applying auth.0006_require_contenttypes_0002... OK
dz01     |   Applying auth.0007_alter_validators_add_error_messages... OK
dz01     |   Applying auth.0008_alter_user_username_max_length... OK
dz01     |   Applying sessions.0001_initial... OK
dz01     | [2018-08-09 11:25:17 +0000] [10] [INFO] Starting gunicorn 19.6.0
dz01     | [2018-08-09 11:25:17 +0000] [10] [INFO] Listening at: http://0.0.0.0:8000 (10)
dz01     | [2018-08-09 11:25:17 +0000] [10] [INFO] Using worker: sync
dz01     | [2018-08-09 11:25:17 +0000] [16] [INFO] Booting worker with pid: 16
dz01     | No Local Settings Found
dz01     | No Local Settings Found
dz01     | /usr/local/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
dz01     |   """)
dz01     | /usr/local/lib/python3.7/site-packages/psycopg2/__init__.py:144: UserWarning: The psycopg2 wheel package will be renamed from release 2.8; in order to keep installing from binary please use "pip install psycopg2-binary" instead. For details see: <http://initd.org/psycopg/docs/install.html#binary-install-from-pypi>.
dz01     |   """)
dz01     | /usr/local/lib/python3.7/site-packages/celery/platforms.py:796: RuntimeWarning: You're running the worker with superuser privileges: this is
dz01     | absolutely not recommended!
dz01     | 
dz01     | Please specify a different user using the --uid option.
dz01     | 
dz01     | User information: uid=0 euid=0 gid=0 egid=0
dz01     | 
dz01     |   uid=uid, euid=euid, gid=gid, egid=egid,
dz01     | [2018-08-09 11:25:18,476: CRITICAL/MainProcess] Unrecoverable error: SyntaxError('invalid syntax', ('/usr/local/lib/python3.7/site-packages/celery/backends/redis.py', 22, 19, 'from . import async, base\n'))
dz01     | Traceback (most recent call last):
dz01     |   File "/usr/local/lib/python3.7/site-packages/kombu/utils/objects.py", line 42, in __get__
dz01     |     return obj.__dict__[self.__name__]
dz01     | KeyError: 'backend'
dz01     | 
dz01     | During handling of the above exception, another exception occurred:
dz01     | 
dz01     | Traceback (most recent call last):
dz01     |   File "/usr/local/lib/python3.7/site-packages/celery/worker/worker.py", line 205, in start
dz01     |     self.blueprint.start(self)
dz01     |   File "/usr/local/lib/python3.7/site-packages/celery/bootsteps.py", line 115, in start
dz01     |     self.on_start()
dz01     |   File "/usr/local/lib/python3.7/site-packages/celery/apps/worker.py", line 139, in on_start
dz01     |     self.emit_banner()
dz01     |   File "/usr/local/lib/python3.7/site-packages/celery/apps/worker.py", line 154, in emit_banner
dz01     |     ' \n', self.startup_info(artlines=not use_image))),
dz01     |   File "/usr/local/lib/python3.7/site-packages/celery/apps/worker.py", line 217, in startup_info
dz01     |     results=self.app.backend.as_uri(),
dz01     |   File "/usr/local/lib/python3.7/site-packages/kombu/utils/objects.py", line 44, in __get__
dz01     |     value = obj.__dict__[self.__name__] = self.__get(obj)
dz01     |   File "/usr/local/lib/python3.7/site-packages/celery/app/base.py", line 1196, in backend
dz01     |     return self._get_backend()
dz01     |   File "/usr/local/lib/python3.7/site-packages/celery/app/base.py", line 914, in _get_backend
dz01     |     self.loader)
dz01     |   File "/usr/local/lib/python3.7/site-packages/celery/app/backends.py", line 70, in by_url
dz01     |     return by_name(backend, loader), url
dz01     |   File "/usr/local/lib/python3.7/site-packages/celery/app/backends.py", line 50, in by_name
dz01     |     cls = symbol_by_name(backend, aliases)
dz01     |   File "/usr/local/lib/python3.7/site-packages/kombu/utils/imports.py", line 56, in symbol_by_name
dz01     |     module = imp(module_name, package=package, **kwargs)
dz01     |   File "/usr/local/lib/python3.7/importlib/__init__.py", line 127, in import_module
dz01     |     return _bootstrap._gcd_import(name[level:], package, level)
dz01     |   File "<frozen importlib._bootstrap>", line 1006, in _gcd_import
dz01     |   File "<frozen importlib._bootstrap>", line 983, in _find_and_load
dz01     |   File "<frozen importlib._bootstrap>", line 967, in _find_and_load_unlocked
dz01     |   File "<frozen importlib._bootstrap>", line 677, in _load_unlocked
dz01     |   File "<frozen importlib._bootstrap_external>", line 724, in exec_module
dz01     |   File "<frozen importlib._bootstrap_external>", line 860, in get_code
dz01     |   File "<frozen importlib._bootstrap_external>", line 791, in source_to_code
dz01     |   File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
dz01     |   File "/usr/local/lib/python3.7/site-packages/celery/backends/redis.py", line 22
dz01     |     from . import async, base
dz01     |                       ^
dz01     | SyntaxError: invalid syntax
dz01 exited with code 1
```

**Comment by mhetreramesh** [comment]
I had to change **python:latest** to **python:3.6** in Dockerfile & it works now.

**Comment by ruddra** [comment]
Thanks for notifying. I have already put a fix at #13
